README.md

```diff
@@ -69,63 +69,34 @@ Please ensure that Verificarlo's dependencies are installed on your system:
   * GNU mpfr library http://www.mpfr.org/
   * LLVM, clang and opt from 3.3 up to 9.0.1, http://clang.llvm.org/
```

Comment:
It is not clear to me which are the minimal requirements for LLVM:
- here it is 3.3
- in autoconf.ac it is 4.0
- below for flang minimal version is 7.0

Could we clarify ?

Reply:
You're right, I forgot to standardize.